src/main/java/com/example/jpa/board/domain/Board.java

```diff
@@ -0,0 +1,59 @@
+package com.example.jpa.board.domain;
+
+import com.example.jpa.board.dto.request.BoardCreateRequest;
+import com.example.jpa.board.dto.request.BoardUpdateRequest;
+import com.example.jpa.member.domain.Member;
+import com.example.jpa.reply.domain.Reply;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Setter
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Board {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "board_id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id")
+    private Member member;
```

Comment:
작성자의 의미라면 writer or author 같은 이름이 더 적절할 것 같음
```suggestion
    private Member writer;
```